doc/nrf/ug_multi_image.rst

```diff
@@ -129,58 +132,88 @@ For example:
 
-When enabling a child image, you select the build strategy, thus how the image is included.
-The three options are:
-
-* Build the child image from source along with the parent image - *IMAGENAME*\_BUILD_STRATEGY_FROM_SOURCE
-* Merge the specified HEX file of the child image with the parent image - *IMAGENAME*\_BUILD_STRATEGY_USE_HEX_FILE, and *IMAGENAME*\_HEX_FILE to specify the HEX file
-* Ignore the child image when building and build only the parent image - *IMAGENAME*\_BUILD_STRATEGY_SKIP_BUILD
+When enabling a child image, you must select the build strategy to define how the image should be included.
+The following three options are available:
 
+* Build the child image from source along with the parent image - ``<IMAGE_NAME>\_BUILD_STRATEGY_FROM_SOURCE``.
+* Merge the specified HEX file of the child image with the parent image - ``<IMAGE_NAME>\_BUILD_STRATEGY_USE_HEX_FILE``, using ``<IMAGE_NAME>\_HEX_FILE`` to specify the HEX file.
+* Ignore the child image when building and build only the parent image - ``<IMAGE_NAME>\_BUILD_STRATEGY_SKIP_BUILD``.
 
 .. note::
 
-   Child images that are built with the build strategy *IMAGENAME*\ _BUILD_STRATEGY_SKIP_BUILD or *IMAGENAME*\ _BUILD_STRATEGY_USE_HEX_FILE must define a :ref:`static partition <ug_pm_static_providing>`.
-
+   Child images that are built with the build strategy ``<IMAGE_NAME>\ _BUILD_STRATEGY_SKIP_BUILD`` or ``<IMAGE_NAME>\ _BUILD_STRATEGY_USE_HEX_FILE`` must define a :ref:`static partition <ug_pm_static_providing>`.
 
-You must add these four configuration options to the Kconfig file for your child image, replacing *IMAGENAME* with the (uppercase) name of your child image (as specified in ``add_child_image``).
+Add these configuration options to the Kconfig file of your child image, replacing ``<IMAGE_NAME>`` with the uppercase name of your child image, as specified in ``add_child_image``.
 
-This can be done by including the :file:`Kconfig.template.build_strategy` template, as shown below.
+This can be done by including the :file:`Kconfig.template.build_strategy` template as follows:
 
 .. code-block:: Kconfig
 
    module=MCUBOOT
    source "${ZEPHYR_NRF_MODULE_DIR}/subsys/partition_manager/Kconfig.template.build_strategy"
 
+.. _ug_multi_image_variables:
 
 Image-specific variables
 ========================
 
 The child image and parent image are executed in different CMake processes and thus have different namespaces.
 Variables in the parent image are not propagated to the child image, with the following exceptions:
 
-* Any variable named *IMAGENAME*\_FOO in a parent image is propagated to the child image named *IMAGENAME* as FOO.
-* Variables that are in the list ``SHARED_MULTI_IMAGE_VARIABLES`` are propagated to all child images.
+* Any variable named ``<IMAGE_NAME>\_FOO`` in a parent image is propagated to the child image named ``<IMAGE_NAME>`` as ``FOO``.
+* Variables that are in the ``SHARED_MULTI_IMAGE_VARIABLES`` list are propagated to all child images.
```

Comment:
How does a user know where to find this list: `SHARED_MULTI_IMAGE_VARIABLES`.

We should not refer to such a list without providing more details.
I think that it should be enough to state:
```suggestion
* CMake build settings, such as `BOARD_DIR`, build type, Toolchain info, Partition manager info, and similar are always passed to child images.
```
@FrancescoSer  please review the text suggestion.